client/rpc/src/chain/mod.rs

```diff
@@ -185,6 +185,44 @@ where
 	) -> RpcResult<bool> {
 		Ok(self.subscriptions().cancel(id))
 	}
+
+	/// Finalized block headers subscription.
+	///
+	/// Returns headers of all blocks that have been finalized, rather than just the latest.
+	fn subscribe_new_finalized_block_headers(
```

Comment:
can we rename to subscribe_all_finalized_heads, and merge with a new tag, something like:

chainflip/monthly-2022-05+1